app/controlplane/pkg/data/ent/migrate/migrations/20241107213854.sql

```diff
@@ -0,0 +1,12 @@
+-- we mark the latest version of each project as prerelease and unset the rest
```

Comment:
I don't understand the query.
```sql
SET prerelease = (id IN (
  SELECT id
  FROM ranked_versions
  WHERE rn = 1
))
```
We are setting the value of `prerelease` to the IDs of the `ranked_version` query? Shouldn't that be on the WHERE clause to filter the values and set the prerelease to something else?

Reply:
it's the other way around, we want to remove the prerelease status from the older ones (since before we culdn't not make them prerelease). Your proposal unset pre-release to the top one.

Does it make sense?